Sofa/Sofa/Header/HeaderView.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  HeaderView.swift
+//  Sofa
+//
+//  Created by Akademija on 04.04.2025..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class HeaderView: BaseView {
```

Comment:
ViewController treba preimovati i napraviti MainEventsViewController ili MainMatches. Treba napraviti grupu (folder) nazvati MainMatches i sve vezano za taj screen/module treba biti tamo.

Pošto se ovaj header nalazi na MainMatchesViewController treba se zvati MainMatchesHeaderView. Jer ovo je sada generali naziv, ako imaš 100 screenova, šta ti znači kad searchaš u projektu i vidiš HeaderView.